pkg/catalog/catalog.go

```diff
@@ -252,7 +252,7 @@ func UpdateStatusWithSyncErr(ctx context.Context, mc model.ClientSet, c *model.C
 		status.CatalogStatusReady.True(c, "")
 
 		// Notify builtin catalog to update builtin resource definitions.
-		if c.Name == "builtin" && settings.EnableBuiltinResourceDefinition.ShouldValueBool(ctx, mc) {
+		if c.Name == "builtin" && settings.EnableBuiltinCatalog.ShouldValueBool(ctx, mc) {
```

Comment:
EnableBuiltinCatalog Setting controls builtin resource definitions and catalogs, shall we have another name of this setting? E.G. `enableBuiltIn` or another reasonable name

Reply:
EnableBuiltin seems more ambiguous. Keep this for now.